BinaryData/Resource/TranslationKeysExtractor.cmake

```diff
@@ -0,0 +1,31 @@
+file(GLOB_RECURSE TranslationSourceFiles
+    ${CMAKE_CURRENT_SOURCE_DIR}/Source/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/Source/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/Dependencies/Misc/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/Dependencies/Misc/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_audio_devices/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_audio_devices/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_gui_basics/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_gui_basics/*.h
+)
+
+set(TRANSLATION_FILE "${CMAKE_CURRENT_BINARY_DIR}/TranslationKeys.txt")
+file(TOUCH ${TRANSLATION_FILE})
```

Comment:
If the target is called several times, the translations will be added to the file. The file should be empty.